bitnami/fluentd/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluentd
-version: 0.5.4
+version: 1.0.0
```

Comment:
Does this really need to be a major?

Reply:
The container will behave differently, that's why I believe it should be major